File: For Fun/Assignment 2/C# version/Palindrome.cs
using System;

class Palindrome
{
	public static void Main(string[] args)
	{
		Console.WriteLine("Enter a line of text: ");
		string input = Console.ReadLine();
		string onlyLetters = "";
		string reverse = "";

		for (int i = 0; i < input.Length; i++)
		{
			if (Char.IsLetter(input[i]))
			{
				onlyLetters += input[i];
			}
		}

		for (int i = onlyLetters.Length - 1; i >= 0; i--)
		{
			reverse += onlyLetters[i];
		}

		Console.WriteLine("Is palindrome: " + reverse.ToLower().Equals(onlyLetters.ToLower()));
	}
}

File: sb223ce_assign2/Palindrome.java
package sb223ce_assign2;
import java.util.Scanner;


public class Palindrome {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		System.out.print("Type a line: ");
		String text = scan.nextLine();
		scan.close();
		String str = "";
		String buf = "";
		
		
		/*First we will use for loop and pick only letter from given line*/
		
		for (int i = 0; i < text.length(); i++) {
			char c = text.charAt(i);

			if (Character.isLetter(c) && Character.isLowerCase(c)) {
				str = str + c;
			} else if (Character.isUpperCase(c)) {
				str = str + (Character.toLowerCase(c));     /*Change to lower case*/
			}

		}

		
		/*Now, we will reverse those letter and save it in another string*/
		for (int i = str.length() - 1; i >= 0; i--) {
			char ch = str.charAt(i);

			buf = buf + ch;

		}

		
	/* Now, compare the both string order   */
		if (buf.equals(str)) {
			System.out.println("The line is palindrome.");
		}

		else {
			System.out.println("The line is not palindrome.");
		}

	}

}
